File: src/main/java/com/mymod/ModBlocks.java
package com.mymod;

import net.minecraftforge.fml.common.registry.GameRegistry;
import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

/**
 * Created by poegem on 7/17/2017.
 */
public class ModBlocks {
    @GameRegistry.ObjectHolder("mymod:block_cool")
    public static BlockCool blockCool;

    @SideOnly(Side.CLIENT)
    public static void initModel()
    {
        blockCool.initModel();
    }
}

File: src/main/scala/mpoegel/locomotion/ModBlocks.scala
package mpoegel.locomotion

import mpoegel.locomotion.blocks.{BlockCool, BlockLumberYard, BlockPaperMill}
import net.minecraftforge.fml.common.registry.GameRegistry
import net.minecraftforge.fml.relauncher.{Side, SideOnly}

/**
  * Created by Matt Poegel on 7/17/2017.
  */
object ModBlocks {
  @GameRegistry.ObjectHolder("locomotion:block_cool")
  var blockCool: BlockCool = _

  @GameRegistry.ObjectHolder("locomotion:block_lumber_yard")
  var blockLumberYard: BlockLumberYard = _

  @GameRegistry.ObjectHolder("locomotion:block_paper_mill")
  var blockPaperMill: BlockPaperMill = _

  @SideOnly(Side.CLIENT)
  def initModels(): Unit =
  {
    blockCool.initModel()
    blockLumberYard.initModel()
    blockPaperMill.initModel()
  }
}
